internal/datastore/mysql/datastore.go

```diff
@@ -669,3 +672,9 @@ func buildLivingObjectFilterForRevision(revision datastore.Revision) queryFilter
 func currentlyLivingObjects(original sq.SelectBuilder) sq.SelectBuilder {
 	return original.Where(sq.Eq{colDeletedTxn: liveDeletedTxnID})
 }
+
+type debugLogger struct{}
+
+func (debugLogger) Print(v ...any) {
+	log.Logger.Debug().CallerSkipFrame(1).Msg(fmt.Sprint(v...))
```

Comment:
taking [a page from the mysql driver](https://github.com/go-sql-driver/mysql/blob/341a5a5246835b2ac4b8d36bb12a9dfad70663f4/errors.go#L40), maybe this should add a field to the log indicating it's from mysql?

Reply:
good idea!